### ISSUE TITLE
Left-Hand Dominant Mode Actions Reveresed

### ISSUE THREAD

**Issue by TecmagDiams** [jrbudda/Vivecraft_116#22]
Some actions are not properly "flipped" in Left-Hand mode. Currently through my playing I found that while using the Left-Hand mappings with the Left-Hand mode on the following inconsistencies exist:

- Sheilds face at the player, the hand would need to rotate 180 degrees. Effectively left hand players have the sheild on the front side of their wrist not the back side.
- Boats have their turning reveresed in roomscale controls. Turning only the left ore results in turning left, which is very jaring.

**Comment by trentech** [comment]
I'm not feeling the love for lefties..like most things in life...any word on a fix here?

**Comment by Techjar** [comment]
Eventually. A bunch of things have to be reworked for it.

**Comment by yoruvo** [comment]
It's a serious detriment to be unable to use the Shield in Hard mode difficulty. Skeletons are very obnoxious to handle without, in VR. Would a bounty/donation accelerate the process here?

**Comment by Techjar** [comment]
The shield still works, the visual is just a bit screwed up.

**Comment by yoruvo** [comment]
That is true, but in multiplayer, and if interested in maintaining any kind of immersion, it's just not really okay to have the shield in your face, stabbing through your character's chest, when you're holding it casually. And immersion is the point of VR, so, y'know.

**Comment by Techjar** [comment]
Yeah, don't worry, we're not just ignoring the issue. I'm gonna try to make a concerted effort to fix left-handed problems for 1.17. I'm just saying it's not totally unusable as it is.

**Comment by ividyon** [comment]
> I'm gonna try to make a concerted effort to fix left-handed problems for 1.17.

The new blog post by jrbudda now says that "no VR changes are planned for 1.17". Is there an update on this?
